.github/workflows/formatting.yaml

```
@@ -6,7 +6,7 @@ on:
 jobs:
   formatting:
     name: Check code formatting
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Linux, X86]
```

```suggestion
    runs-on: [Linux, X64]
```

For the others, I'd also suggest to use `[Linux, X64]` instead of `explorer` as it is more specific on what the job needs, i.e. it will get scheduled on any similar runners (if we add them).